### ISSUE TITLE
delegate method that returns date only once the animation has finished

### ISSUE THREAD

**Issue by thomasbaldwin** [Mozharovsky/CVCalendar#125]
It would be nice to have a delegate method that returns the date only once the animation has completed. That way I don't start working with a date and call tableView.reloadData along the way, and the animation is cancelled by that action

**Comment by hffmnn** [comment]
I added a pull request for that, see https://github.com/Mozharovsky/CVCalendar/pull/137. Hope it helps.

**Comment by thomasbaldwin** [comment]
Thanks @hffmnn !

**Comment by Mozharovsky** [comment]
@thomasbaldwin, @hffmnn: Guys, if this issue isn't solved, let me know, I'll reopen the thread.
